### ISSUE TITLE
FEATURE: Better images preview on the wall

### ISSUE THREAD

**Issue by acs-ferreira** [humhub/humhub#2351]
Actually images displayed on the wall as previews are not optimal, even worse if images have different sizes.

How about to introduce a tiny jQuery plugin `flexImages` via bower to optimize that?
https://github.com/Pixabay/jQuery-flexImages
https://goodies.pixabay.com/jquery/flex-images/demo.html

**Comment by Felli** [comment]
[flexImage](https://github.com/Pixabay/jQuery-flexImages#features) doesn't have the possibly of video gallery it would seem? Blueimp has this implemented already which will be helpful once a video player is implemented later for video galleries. To be honest I myself liked [lightGallery](http://sachinchoolur.github.io/lightGallery/) but Blueimp isn't that bad. :smile:

**Comment by acs-ferreira** [comment]
It supports videos, and its purpose is to create responsive columns, not to provide the Blueimp functions.

So its flexImages + Blueimp.

**Comment by buddh4** [comment]
looks nice, I'll try this if I find the time.

**Comment by acs-ferreira** [comment]
@buddh4 any chances to have this on 1.3?

**Comment by Carlos2213** [comment]
Agreed. The gallery module is terrible. It lacks a grid layout and more importantly, the pictures are not searchable. Not by caption or by file name. 

But not using the gallery is even worse, because photos are not even grouped in any way without it.

**Comment by Abatjour3407** [comment]
Hi, I'm not sure if this is relevant to this feed but when attaching images to a wall/feed post on HumHub it doesn't have a nice impact (let's compare it to how it's presented on other social media platforms): It displays these images as little thumbnails in columns instead of having a full-width appearance. I think this should be an easy thing to fix and I've fixed it on my own platform in a mobile responsive way but I'm not an expert developer so I know someone can do a better job of it than me.

**Comment by buddh4** [comment]
@Abatjour3407 maybe add a screenshot of your prefered solution.

**Comment by Abatjour3407** [comment]
The importance of this isn't so evident in this image, but when uploading an advertisement or call to action image it loses all of it's impact. Maybe when adding multiple images the current way works best but I think for a single image this can be improved:
![SUGGESTION](https://user-images.githubusercontent.com/50051640/108076340-fe9a3100-7073-11eb-8c82-0508b38600ad.png)
`

**Comment by buddh4** [comment]
I think on mobile we already have an option to use a slider like image gallery. This needs to be activated (not sure if its active by default) `Administration -> Settings -> Appearance`. But this should only work on mobile devices. I think on desktop a tile like presentation would be the best (but a more complex one as currently).

**Comment by Abatjour3407** [comment]
Ok, thanks. I can see the merits to your argument. 

I'll keep my customisations as full width on desktop with maybe a screen-height related max-width but as for the main HumHub product, that's not my choice to make. Just thought to bring your attention to the possible variation. 

Thanks for the info on the mobile appearance and thanks for engaging with this idea. Have an epic day further that side :)
